src/main/java/com/microsoft/spring/data/gremlin/query/GremlinOperations.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.query;
+
+/**
+ * Provider interface for basic Operations with Gremlin
+ */
+public interface GremlinOperations {
+
+    void deleteAll();
+
+    <T> T insert(T object);
```

Comment:
The generics is used in the parameter but not for the class, does that mean this `GremlinOperations ` can insert whatever type?

Reply:
yes, for most data framework, any instanced class can be entity here.